pkg/agent/interfacestore/types.go

```diff
@@ -43,6 +43,7 @@ const (
 	AntreaUplink           = "uplink"
 	AntreaHost             = "host"
 	AntreaTrafficControl   = "traffic-control"
+	AntreaIPsec            = "ipsec"
```

Comment:
perhaps ipsec-tunnel to be more specific.

Reply:
Done.